app/views/sales/current_month.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="sub-header">
+  <h3>Current Month's Sales for <%= @vendor.name %></h3>
+</div>
+<% total_sales = [] %>
+<% time_marker_month = Time.now.month %>
+<% time_marker_year = Time.now.year %>
+<table>
+  <% @sales.each do |s| %>
+    <% if s.purchase_time.month == time_marker_month && s.purchase_time.year == time_marker_year %>
+      <% total_sales.push(s.amount) %>
+      <tr>
+        <td>
+          $<%= p '%.2f' % (s.amount/100.0) %>
+        </td>
+        <td>
+          <%= s.purchase_time.strftime("%D") %>
+        </td>
+      </tr>
+    <% end %>
+  <% end %>
+      <tr>
```

Comment:
This section from here through line 30 should be de-indented so that the `<tr>` opening and closing tags are on the same indentation level as the `<% end %>` from the line above.